tests/server/flow-event.js

```diff
@@ -60,6 +58,18 @@ registerSuite('flow-event', {
       assert.lengthOf(flowEvent, 3);
     },
 
+    'call flowEvent with flow_metrics_disabled true': {
+      beforeEach() {
```

Comment:
Same as above, you will need to actually try to log a flowEvent and test that nothing was output. See `call flowEvent with valid flow data` test for an example